### DISCUSSION TITLE
AI didnt detect children

### DISCUSSION THREAD

**Discussion by Captainslam** [photoprism/photoprism#4220] [Q&A]
Wanted to give this a go for my wife as an alternative of Google. Just loaded 50k photos and of all the faces it did detect, it didnt for our kids which is a massive amount of these photos. Am I missing something? I did the reindex again after the initial photo import

**Comment by Deldion** [comment]
What I've witnessed is that kid's faces get mixed up. And they do change over time so it must be hard I guess?

**Comment by poisonnuke** [comment]
It did detect my toddlers but the facial recognition itself is just crap. On average only 30% of all faces in a picture are even detect as a face. Even high quality portraits are not detected. 

Incorrect or not working tagging is the smallest issue,

The AI engine needs a massive overhaul. 30% does not qualify for any released product, especially when other services achieve > 99%.

**Reply by lastzero** [reply]
It's a free feature and you are welcome to contribute to make it better.

**Reply by poisonnuke** [reply]
There is only one realistic solution: immich has the by far better feature for facial recognition and some other things, this soloution is superior in terms of UI and search and map (even though there is a lot to do).
Why not working together with immich and join to one project?

**Reply by lastzero** [reply]
We have a lot of happy users who love our software, so we won't let them down. Clearly that's not you, and we're not the Google Photos clone you're looking for. Since you don't seem interested in contributing or using our software, let's just leave it at that.

**Reply by poisonnuke** [reply]
you did not read my message though, as it cleary stated the exackt (!) opposite of that. 
I will not leave it with that...

**Reply by poisonnuke** [reply]
ok lets take some more time, my last posting was a little harsh but for a good reason:
users are constantly ignored around here, Ive read a lot here and there are so many places were general ideas of users are just played down or ignored.

There is especially one major misconception that bugs me: the userbase of this application is _not_ the average Joe. The average Joe is not able to setup and maintain a docker-installation, and the average Joe does not spend a lot of time to find a good solution to organize pictures. 

Users that "end up" with photoprism or immich are "professional amateurs", they already spent a great deal of time to find a solution to organize their pictures and generally they already have an scheme how to organize pictures.
Photoprism and immich on the other hand just a plain copy Google Photos with all the drawbacks (no fine-grain control, no levels of organization. Its just a flat list of pictures without any context). 

A lot of users are requesting a rethinking of the general strategy to differ from Google Photos (because thats what the average Joe is doing, they just use Google Photos or iCloud, they dont care about self-hosting something). So because >>90% of your users want professional features like multiple levels of hierarchy, better face-tagging, better labeling and so on, some rethinking on your end must happen.

And what I was proposing is actually a valid idea: immich and photoprism are currently the only photo-solutions that one can seriously consider. But both have their strenght and a lot of weaknesses. So as of now there is not a single solution that one can use.

This is the culprit of getting contribution: Im not willing to contribute to a project that is stuck in a loop of "we are already good and dont need to improve the base". And Im especially not willing to contribute to a project that is not accepting the opinions of its users. In the end I want to use ONE photo-library that will life for a longer time and there is no way that immich and photoprism will coexist forever, not if there is not a big change in either one of them.


And whats also  a valid point from my end in the previous posting: choosing a solution that is know to fail, is actually a bad decision. Pigo is just crap and should not have been used in the first place (with a proven detection rate of less than 10% its really not useful at all)

How am I supposed to contribute with such major roadblocks?

**Reply by Deldion** [reply]
You just get in there and get your hands dirty?
You devote your time and you give it your own twist? You propose and contribute?

I do get your point. Better one "robust" solution than two "weaker" ones, but it's the same as the miriads of Linux distros.
It's open for anyone to interpret and play with!

So you give your precious time, if you have any to spare, like all these people that do give their time to deliver an alternative to the dystopia of having large corporations scanning your toddlers' faces. 

Am I missing something?

**Reply by poisonnuke** [reply]
> You just get in there and get your hands dirty? You devote your time and you give it your own twist? You propose and contribute?

> Am I missing something?

yes, the most important thing: its not about doing a small PR here and there. Its about the overarching mindset. Why pretenting to do something else then Google Photos when the core of the application is essentially the same.
And why pretending the user-base is happy and the user-base is the average Joe, when this is clearly not the case. The current focus is on (bluntly speaken) irrelevant features that have quite a small impact but require a great deal of story points.

There must be a change in where the project itself is heading to, which is nothing someone can just "contribute", thats something that the core-team has to discuss and plan. My contribution is to provide and overarching outside view that is quite comprehensive, but I get the feeling thats not being wished for.
